app/src/main/java/com/sunasterisk/youtubebackground/utils/VideoPlayService.kt

```diff
@@ -0,0 +1,116 @@
+package com.sunasterisk.youtubebackground.utils
```

Comment:
service đặt trong package service 
```suggestion
package com.sunasterisk.youtubebackground.service
```